### ISSUE TITLE
Error running Angular Signature Sample

### ISSUE THREAD

**Issue by tarekahf** [legalthings/angular-signature#29]
When working with the simple example, I get this error as soon as the page is loaded:

```
VM697 angular.min.js:103 Error: [$compile:nonassign] http://errors.angularjs.org/1.3.8/$compile/nonassign?p0=undefined&p1=signaturePad
    at Error (native)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:6:416
    at l (https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:58:364)
    at f (https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:58:441)
    at Object.<anonymous> (https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:109:503)
    at l.$digest (https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:123:310)
    at l.$apply (https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:126:362)
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:17:448
    at Object.e [as invoke] (https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:37:96)
    at d (https://ajax.googleapis.com/ajax/libs/angularjs/1.3.8/angular.min.js:17:369)
```

and when I draw the signature, I see this error in the console:

`VM697 angular.min.js:103 TypeError: f is not a function`

However, the signature is working, and the Data URL is being retrieved.

Check sample here:

http://plnkr.co/edit/TvpmtxjulMFHtTCR6SnZ?p=preview

Press F12 to see the console errors.

Is there something wrong with my code?

Tarek

**Comment by jasny** [comment]
The first error is solved. A number of attributes ware required that should have been optional.

The `f is not a function` error still remains.

**Comment by tarekahf** [comment]
I reviewed the demo here:

https://rawgit.com/legalthings/angular-signature/master/demo/index.html

Then I ensured the script references are correct in my Plunker sample here:

http://plnkr.co/edit/TvpmtxjulMFHtTCR6SnZ?p=preview

Now, seems like it is working fine.

It appears that you have fixed the bug in a recent version, or I was using wrong references in my old Plunker sample.

Thank you anyway. However, in my project, I used ng-signature-pad here:

https://github.com/ecentinela/ng-signature-pad#ng-signature-pad

Tarek
